docs/deploying.md

```diff
@@ -0,0 +1,39 @@
+Deploying perf.html
```

Comment:
All the other docs use `#` style headers, so it might be better to be consistent in the markdown style we are using.

Reply:
OK I'll change these, thx